modules/installation-vsphere-machines.adoc

```diff
@@ -0,0 +1,142 @@
+----
+
+. In the vSphere Client, create a folder in your datacenter to store your VMs.
+.. Click the *VMs and Templates* view.
+.. Right-click the name of your datacenter.
+.. Click *New Folder* -> *New VM and Template Folder*.
+.. In the window that is displayed, enter the folder name. The folder name must
+match the cluster name that you specified in the `install-config.yaml` file.
+
+. In the vSphere Client, create a template for the OVA image.
++
+[NOTE]
+====
+In the following steps, you use the same template for all of your cluster
+machines and provide the location for the Ignition config file for that machine
+type when you provision the VMs.
+====
+.. From the *Hosts and Clusters* tab, right-click your cluster's name and
+click *Deploy OVF Template*.
+.. On the *Select an OVF* tab, specify the name of the {op-system} OVA file
+that you downloaded.
+.. On the *Select a name and folder* tab, set a *Virtual machine name*, such
+as {op-system}, and click the name of the your vSphere cluster.
+.. On the *Select a compute resource* tab, click the name of the your vSphere
+cluster.
+.. On the *Select storage* tab, configure the storage options for your VM.
+*** Select *Thin Provision*.
+*** Select the datastore that you specified in your `install-config.yaml` file.
+.. On the *Select network* tab, specify the the network that you configured
+for the cluster, if available.
+.. If you plan to use the same template for all cluster machine types, do not
+specify values on the *Customize template* tab.
+
+. After the template deploys, deploy a VM for a machine in the cluster.
+.. Right-click the template's name and click *Clone* -> *Clone to Virtual Machine*.
+.. On the *Select a name and folder* tab, specify a name for the VM. You might
+include the machine type in the name, such as `control-plane-0` or `compute-1`.
+.. On the *Select a name and folder* tab, select the name of the folder that
+you created for the cluster.
+.. On the *Select a compute resource* tab, select the name of a host in your
+datacenter.
+.. Optionally, on the *Select storage* tab, customize the storage options.
+.. On the *Select clone options*, select
+*Customize this virtual machine's hardware*.
+.. On the *Customize hardware* tab, click *VM Options* -> *Advanced*.
+*** From the *Latency Sensitivity* list, select *High*.
+*** Click *Edit Configuration*, and on the *Configuration Parameters* window,
+click *Add Configuration Params*. Define the following parameter names and values:
+**** `guestinfo.ignition.config.data`: Paste the contents of the base64-encoded
+Ignition config file for this machine type.
+**** `guestinfo.ignition.config.data.encoding`: Specify `base64`.
+**** `disk.EnableUUID`: Specify `true`.
+.. In the *Virtual Hardware* panel of the
+*Customize hardware* tab, modify the specified values as required. Ensure that
+the amount of RAM, CPU, and disk storage meets the minimum requirements for the
+machine type.
```

Comment:
And after 'Add Configuration Params', in the follow wizard - 'Customize vApp properties', user need input 'guestinfo.ignition.config.data.encoding' and 'guestinfo.ignition.config.data.encoding' again. This is really important. We need mention the step here.

Reply:
@jianlinliu That is not necessary. The user needs to either specify the properties from the Configuration Parameters or from the Configure tab. They do not need to do both.

Reply:
@staebler, @jianlinliu  should we consider it to be a usability enhancement to add the the parameters to the VMs after initialization instead of setting them earlier? I can change the docs to just add them here.

Reply:
I find it to be a moderately better user experience to use the Configure/vApp Properties UI. However, since the user already has to go to the Advanced/Configuration Properties UI to enable disk UUID, I don't have a strong opinion about which way we should guide the user. I do think that we should only guide them one way and not both.

Reply:
I agree that we should only describe one set of actions, and if we didn't need to go enable disk UUID, this method would win. I vote to leave it as-is for now.